submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,382 @@
+ caption='Processing coal into solid fuel with heavy oil cracking, at minimum power consumption'
+
+%}
+
+Now we recalculate the energy gain with the decreased power consumption:
+
+`Energy gain = 88.5 - 40 - 1.5 - 0.42 - 0.1365 - 0.5775 - 0.042 = 45.824 MJ`
+
+`Percent gain = 45.824 MJ / 40 MJ = 114.56% ≈ 115%` 
+
+The energy gain was 103% with no modules. We see that efficiency modules increase the energy gain from 103% to 115% This increase is nice, but it is not that significant, since the processing energy cost was already relatively low. Nevertheless, efficiency 1 modules being cheap to produce makes this option worthwhile. 
+
+Now, let us look at the same example with three productivity 3 modules in every machine instead of efficiency 1 modules. Despite increasing the power consumption they also increase the amount of fuel produced, by 10% per machine per module:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at maximum productivity'
+ caption='Processing coal into solid fuel with heavy oil cracking, at maximum productivity'
+
+%}
+
+We see that with the cumulative effects of 30% extra product output across the process results in more than double the amount of solid fuel as before! This adds up to an extra energy gain of nearly 100 MJ! However, we also see that total power consumption has greatly increased to about 50.6 MJ.  Let's see how all these changes reflect to the energy gain: 
+
+`Energy gain = 185.562 - 40 - 1.5 - 12.982 - 5.9716 - 30.040 - 1.688 = 93.3804 MJ`
+
+`Percent gain = 93.3804 MJ / 40 MJ = 233.451% ≈ 233%`
+
+We find that productivity modules increase energy gains from 103% to 233%, meaning that in this setup the solid fuel being produced has **over triple the energy** as the coal used! 
+
+On the other hand, our machines are working at about half the speed as before, which makes power get consumed for twice as long. Perhaps we can use speed modules to counteract this. Let's try switching to two productivity 3 modules and one speed 3 module for each machine:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing coal into solid fuel with heavy oil cracking, at high productivity & speed'
+ caption='Processing coal into solid fuel with heavy oil cracking, , at high productivity & speed'
+
+%}
+
+Compared to the last setup, we now get less than double the solid fuel but the power consumption has approximately halved. Let's see how it reflects to the energy gain: 
+
+`Energy gain = 146.448 - 40 - 1.5 - 5.775 - 2.3966 - 11.400 - 0.693 = 84.6834 MJ`
+
+`Percent gain = 84.6834 MJ / 40 MJ = 211.7085% ≈ 212%`
+
+Compared to no modules offering 103% gain, this setup offers an excellent gain of 212%, but the full-productivity setup with 233% gain is better. This is because the cumulative fuel gains from the third productivity 3 module outweigh the reduced power consumption from the speed 3 module. 
```

Comment:
```suggestion
Compared to no modules offering 103% gain, this setup offers an excellent gain of 212%, but the full-productivity setup with 233% gain is better. This is because the cumulative fuel gains from the third productivity 3 module outweighs the reduced power consumption from the speed 3 module.
```

Reply:
Upon discussion, I decided to keep it unchanged.